packages/nuxt/src/app/plugins/check-outdated-build.client.ts

```diff
@@ -4,16 +4,18 @@ import type { NuxtAppManifestMeta } from '../composables/manifest'
 import { onNuxtReady } from '../composables/ready'
 // @ts-expect-error virtual file
 import { buildAssetsURL } from '#internal/nuxt/paths'
+// @ts-expect-error virtual file
+import { outdatedBuildInterval } from '#build/nuxt.config.mjs'
 
 export default defineNuxtPlugin((nuxtApp) => {
-  if (import.meta.test) { return }
+  if (import.meta.test || !outdatedBuildInterval) { return }
```

Comment:
I think if it's set explicitly to `false` we can completely omit adding this plugin, here:

https://github.com/nuxt/nuxt/blob/5b1ca73886d942db634fc8f939a4ee3db16e47f1/packages/nuxt/src/core/nuxt.ts#L502